url-updater/src/main/java/com/devonfw/tools/ide/url/tool/intellij/IntellijUrlUpdater.java

```diff
@@ -7,6 +7,9 @@
 import com.devonfw.tools.ide.json.JsonVersionItem;
 import com.devonfw.tools.ide.os.OperatingSystem;
 import com.devonfw.tools.ide.os.SystemArchitecture;
+import com.devonfw.tools.ide.tool.intellij.IntellijJsonDownloadsItem;
+import com.devonfw.tools.ide.tool.intellij.IntellijJsonObject;
+import com.devonfw.tools.ide.tool.intellij.IntellijJsonRelease;
```

Comment:
Shouldn't we also move these files here?
They are IMHO only used by the url-updaters and not by IDEasy and URL-Metadata.

Reply:
Moved to url-updater.

Reply:
same here. Still in cli module.